AllisonsCatPalaceTDDTests/FavoritesListControllerTests.swift

```diff
@@ -44,24 +44,14 @@ class CatListControllerTests: XCTestCase {
     }
 
     func testIsTableViewController() {
-        XCTAssert(controller as Any is UITableViewController, "CatListController should be a UITableViewController")
+        XCTAssert(controller as Any is UITableViewController, "FavoritesListController should be a UITableViewController")
     }
 
-    func testHasNoCatsByDefault() {
-        XCTAssert(controller.cats.isEmpty, "CatListController should have no cats by default")
+    func testHasNoAnimalsByDefault() {
+        XCTAssert(controller.animals.isEmpty, "FavoritesListController should have no animals by default")
     }
 
-    func testViewDidLoadCallsSuperViewDidLoad() {
```

Comment:
can leave in since will pass automatically unless I screwed up the implementation.